feat: update readme (#2)
reearth/reearth: README.md

[![stars](https://img.shields.io/github/stars/reearth/reearth)](https://img.shields.io/github/stars/reearth/reearth)
[![GitHub stars](https://img.shields.io/github/stars/reearth/reearth.svg?style=social&label=Star&maxAge=2592000)](https://github.com/reearth/reearth/stargazers/)
[![issues](https://img.shields.io/github/issues/reearth/reearth)](https://img.shields.io/github/issues/reearth/reearth)
[![forks](https://img.shields.io/github/forks/reearth/reearth)](https://img.shields.io/github/forks/reearth/reearth)
[![](https://img.shields.io/github/license/reearth/reearth)](https://img.shields.io/github/license/reearth/reearth)

> [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/reearth/reearth/graphs/commit-activity)

Reviewer: Not sure why but "Maintained?" tag is showing twice. Also not sure why here it says "Maintenance" and on the page is says "Maintained", but if possible can you lowercase them? Everything else is lowercase so unless it depends on the tag's source it'll be better!

Author: I'll delete one. And lowercased. Thanks.

[![GitHub tag](https://img.shields.io/github/tag/reearth/reearth.js.svg)](https://GitHub.com/reearth/reearth.js/tags/)
[![Github all releases](https://img.shields.io/github/downloads/reearth/reearth.js/total.svg)](https://GitHub.com/reearth/reearth.js/releases/)
[![GitHub contributors](https://img.shields.io/github/contributors/reearth/reearth.svg)](https://GitHub.com/reearth/reearth/graphs/contributors/)
[![](https://img.shields.io/github/contributors/reearth/reearth-web?label=front-end%20contributors)](https://GitHub.com/reearth/reearth-web/graphs/contributors/)
[![](https://img.shields.io/github/contributors/reearth/reearth-backend?label=back-end%20contributors)](https://GitHub.com/reearth/reearth-backend/graphs/contributors/)
[![Generic badge](https://img.shields.io/badge/discussion-welcome-green.svg)](https://github.com/reearth/reearth/discussions)

<!-- PROJECT LOGO -->
<br />
<p align="center">
<!-- TODO: change this link later -->
  <a href="https://github.com/reearth/reearth">
    <img src="./public/reearth-logo.svg" alt="Logo" width="300" height="300">
  </a>

  <p align="center">

>     Re:Earth is a state-of-the-art WebGIS with expandability through plug-ins and the ease of running in a browser.

Reviewer: ```suggestion
    Re:Earth is a state-of-the-art WebGIS platform with expandability through plug-ins and has the ease of running in the browser.
```

    <br />
    <a  href="https://docs.reearth.io/"><strong>Documents >></strong></a>
    &emsp;&emsp;&emsp;&emsp;
    <a href="https://docs.reearth.io/oss"><strong>OSS Webpage >></strong></a>
    <br />
    <!-- TODO: here -->
    <a href="">View Demo</a>
    ·
    <a href="https://github.com/reearth/reearth/issues">Report Bug</a>
    ·
    <a href="https://github.com/reearth/reearth/discussions">Request Feature</a>
  </p>
</p>

> Re:Earth consist with 2 repositories. Don't forget to check our [front-end](https://github.com/reearth/reearth-web) and [back-end](https://github.com/reearth/reearth-backend) repositories👍

Reviewer: ```suggestion
Re:Earth consist of 2 repositories. Don't forget to check out our [front-end](https://github.com/reearth/reearth-web) and [back-end](https://github.com/reearth/reearth-backend) repositories 👍
```

[![reearth-ui](./public/reearth-ui.png)](https://reearth.io)

<!-- ABOUT THE PROJECT -->
## Features

> 🔌 Higy extensible with plugins system

Reviewer: ```suggestion
🔌 Highly extensible thanks to the robust plugin system
```

> 💻 Super handy with running on the browser

Reviewer: ```suggestion
💻 Super handy being browser-based
```

> 💪 Support standard GIS data formats(CSV, KML, CZML, GeoJSON, shapefile)

Reviewer: ```suggestion
💪 Supports most standard GIS data formats (CSV, KML, CZML, GeoJSON, shapefile)
```

> 📢 Easily make the project public

Reviewer: ```suggestion
📢 Easily make a project public
```

> ✨ Freely style the map

Reviewer: If this is about being able to add markers and such perhaps this is better:
```suggestion
✨ Freely populate the map
```
If this is about being able to customize the map, like different tiles/atmosphere/etc, the original should be okay

Author: I meant customizing the map. Thanks!

## Built With
* [React](https://github.com/facebook/react)
* [TypeScript](https://github.com/microsoft/TypeScript)
* [Go](https://github.com/golang/go)
* [GraphQL](https://github.com/graphql)
* [MongoDB](https://www.mongodb.com/)
* [WebAssembly](https://webassembly.org/) + [QuickJS](https://github.com/bellard/quickjs)

> ### Prerequisites

Reviewer: ```suggestion
### Necessities
```

**Front-end**

> * Node.js: Latest version of LTS

Reviewer: ```suggestion
* Node.js: Latest LTS version
```

**Back-end**
* Golang +1.16
* Docker

<!-- USAGE EXAMPLES -->
## Usage

> _For more examples, please refer to the [Documentation](https://docs.reearth.io)_

Reviewer: Unless we show an example/examples here, we should just say 
```suggestion
_For examples and more, please refer to the [Documentation](https://docs.reearth.io)_
```

<!-- ROADMAP -->
## Roadmap

> See the [Roadmap](https://github.com/reearth/reearth/projects/1) for a list of proposed features (and known issues).

Reviewer: If the roadmap has proposed features, the features have only been suggested and not committed to being developed. If that is what you mean this is fine.
But if the features listed are on track to being developed (for sure, or 90% sure :P) this might be better:
```suggestion
See the [Roadmap](https://github.com/reearth/reearth/projects/1) for a list of planned features and known issues.
```

Author: The planned proposal will be put in the `Q? 202?` column. The good proposals which are not sure when developed are in `Influential proposals`. In this case, it contains both. Which is better?

> If your have any requests for the features, we are welcome! Just create issues [here](https://github.com/reearth/reearth/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) with `feature request` tag.

Reviewer: ```suggestion
If your have any requests for features, they are more than welcome! Just create an issue [here](https://github.com/reearth/reearth/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) with the `feature request` tag.
```

<!-- CONTRIBUTING -->
## Contributing
### Contributing Guide
<!-- TODO: add link later -->

> Read our [contributing guide]() to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes to React.

Reviewer: ```suggestion
Read our [contributing guide]() to learn about our development process, how to propose bugfixes and improvements, and how to build and test your changes in React.
```

### Code of Conduct

> Re:Earth has adopted a Code of Conduct that we expect project participants to adhere to. Please read [the full](./CODE_OF_CONDUCT.md) text so that you can understand what actions will and will not be tolerated.

Reviewer: ```suggestion
Re:Earth has adopted a Code of Conduct that we expect project participants to adhere to. Please read the [full text](./CODE_OF_CONDUCT.md) so that you can understand what actions will and will not be tolerated.
```

### Good First Issues

> To help you get your feet wet and get you familiar with our contribution process, we have a list of good first issues that contain bugs which have a relatively limited scope. This is a great place to get started.

Reviewer: Where can I find this list of good first issues?

Author: I added the link. thank you.

<!-- LICENSE -->
## License

Distributed under the MIT License. See `LICENSE` for more information.

<!-- CONTACT -->
## Contact

Re:Earth team - info@reearth.dev

<!-- TODO: replace here -->

> Project Link: [https://github.com/reearth/reearth](https://github.com/reearth/reearth)

Reviewer: ```suggestion
Project link: [https://github.com/reearth/reearth](https://github.com/reearth/reearth)
```

* Docker

<!-- USAGE EXAMPLES -->
## Usage

_For examples and more, please refer to the [Documentation](https://docs.reearth.io)_

## Environment
### OS

| Windows10 | Apple macOS 10.12(macOS Sierra)+ | ChromeOS | iOS 11+ | Android 10+ | Linux(which installed browsers blow) |
| --------- | --------- | --------- | --------- | --------- | --------- |
| ✅ | ✅ | ✅ | ✅ | ✅ | ✅ |

### Browser
| ![Edge](https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_32x32.png) <br />Edge | ![Firefox](https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_32x32.png) <br /> Firefox | ![Chrome](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_32x32.png) <br /> Chrome | ![Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_32x32.png) <br /> Safari | ![iOS Safari](https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari-ios/safari-ios_32x32.png) <br />iOS Safari | ![Chrome for Android](https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_32x32.png) <br/> Chrome for Android |
| --------- | --------- | --------- | --------- | --------- | --------- |

> | 91+ | 57+| 58+| 11+ | last 2 versions] | last 2 versions

Author: @KaWaite 
I've added this part. Are `Environment`, `OS`, `Browser` OK? If it's incorrect, I can't have confidence in my English haha.
